app/apps/server/bridges/activation.js

```diff
@@ -16,6 +20,19 @@ export class AppActivationBridge {
 	}
 
 	async appStatusChanged(app, status) {
+		const { _id, username } = Users.findOne({ appId: app.getID() });
```

Comment:
please always use a model function instead of writing your own query/operation. Having all query/operations inside the model helps a lot debugging and specially detecting if a new index is needed.

if there is no method to do what you need in the model already, please create a new one.. in this case I'd say to create something like: `Users.findOneByAppId(app.getID());`

Reply:
Got it! Thanks!